### ISSUE TITLE
`predicate_type` slot is not found anywhere

### ISSUE THREAD

**Issue by gouttegd** [mapping-commons/sssom#404]
The SSSOM LinkML model defines a `predicate_type` slot:

```yaml
slots:
  […]
  predicate_type:
    description: The type of entity that is being mapped.
    range: entity_type_enum
    examples:
      - value: owl:AnnotationProperty
      - value: owl:ObjectProperty
  […]
```

This slot is presumably intended to allow a mapping author to specify whether the nature of the predicate (for example, is it an annotation property or an object property? an information that can be useful to provide as it impacts [how the mapping is to be rendered in OWL](https://mapping-commons.github.io/sssom/spec-formats-owl/)). (As an aside, the _description_ of that slot is misleading, as the slot is not about the “entity that is being mapping“, it is about the predicate that maps the two entities being mapped to each other.)

But that slot is _not_ listed among the allowed slots of the `Mapping` class:

```yaml
classes:
  […]
  mapping:
    description: Represents an individual mapping between a pair of entities.
    slots:
    - subject_id
    - subject_label
    - subject_category
    - predicate_id
    - predicate_label
    - predicate_modifier
    - object_id
    - object_label
    - object_category
    - mapping_justification
    - author_id
    - author_label
    - reviewer_id
    - reviewer_label
    - creator_id
    - creator_label
    - license
    - subject_type
    - subject_source
    - subject_source_version
    - object_type
    - object_source
    - object_source_version
    - mapping_provider
    - mapping_source
    - mapping_cardinality
    - mapping_tool
    - mapping_tool_version
    - mapping_date
    - publication_date
    - confidence
    - curation_rule
    - curation_rule_text
    - subject_match_field
    - object_match_field
    - match_string
    - subject_preprocessing
    - object_preprocessing
    - similarity_score
    - similarity_measure
    - see_also
    - issue_tracker_item
    - other
    - comment
```

In fact, that slot is not referenced _anywhere_ in the entire schema!

**Comment by gouttegd** [comment]
(Can’t help but feeling amused that an issue about a slot not being found anywhere got the issue number **404** … :D )

**Comment by gouttegd** [comment]
Fixed by #405.
